app/frontend/styles/components/vacancy_form.scss

```diff
@@ -0,0 +1,58 @@
+@media (min-width: 40.0625em) {
+  div.no-margin-bottom {
+    .govuk-button {
+      margin-bottom: 0;
+    }
+  }
+
+  div.continue {
```

Comment:
should definatley not use div here but make sure the selector is specific enough to target the element you want `.continue`

Reply:
this wouldnt pass sass linting when we have it in place. at some stage ill be doing a little dev 101 on SASS as frontend wise its the part of the codebase needs some love

Reply:
I need to apply styling to the button, but the form builder abstracts away from it such that you can't apply classes or ids directly to the button. I wonder if you have any ideas about a Noble Third Way that I could try to target the element, without using a useless div?